contour/Dockerfile

```diff
@@ -15,7 +15,7 @@ WORKDIR /work/contour/
 RUN GO111MODULE=on CGO_ENABLED=0 GOOS=linux GOLDFLAGS="-w -s" go build -o /go/bin/contour -v github.com/projectcontour/contour/cmd/contour
```

Comment:
Since CoreDNS runs on Ubuntu, `CGO_ENABLED=0` for static binary is no longer needed.